package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
thanks for the reply @valeriy-devpronet . right thats basics of semver. complexity adds additional conditions when you are under `1.0.0`. So in case of `0.x.y` - `x` is major and `y` is minor as @19majkel94 said.